src/arch/asm.h

```diff
@@ -1,4 +1,4 @@
-#if defined(APPLE) || (defined(WINDOWS) && defined(X86))
```

Comment:
32 bit Windows does use additional leading `_`, you'll have to revert this change.